src/plonk/mod.rs

```diff
@@ -132,6 +132,7 @@ impl<F: PrimeField> CompressedGates<F> {
 pub struct PlonkStructure<F: PrimeField> {
     /// k is a parameter such that 2^k is the total number of rows
     pub(crate) k: usize,
+    // TODO #329 Remove this field
```

Comment:
In current version, the instance we actually pass in is vector of length num_io. Since now we have multiple instance columns, we need something like this:
num_io: Vec<usize> to indicate the actually number of public io for each instance columns. Later, we can concatenate them together with witness into one vector.
I suggest we change this field to type `Vec<usize>` and maybe rename.  Though, the current name is okay to me

Reply:
you can change TODO here if plan to add this info in next PR.

Reply:
Chao, I'm working on these PRs right now, there's not much point in doing an exact TODO, I'll have to handle the instance columns here, however, this field won't exist in this form